services/github/src/api.js

```diff
@@ -89,6 +89,21 @@ function getReleaseDetails(eventData) {
   };
 }
 
+// See https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#check_run
+function getRerunDetails(eventData) {
+  return {
+    'event.type': 'rerun',
+    'event.head.user.login': eventData.sender.login,
+    'event.head.user.id': eventData.sender.id,
+    'event.check.name': eventData.check_run.name,
+    'event.check.run.id': String(eventData.check_run.id),
+    'event.check.run.url': eventData.check_run.url,
+    'event.check.suite.url': eventData.check_run.check_suite.url,
+    'event.check.suite.id': String(eventData.check_run.check_suite.id),
+    'event.head.repo.name': eventData.repository.name,
+  };
+}
```

Comment:
Are all of these used? I think this pattern was because these keys are exposed to the user in the v0 taskcluster.yml.  Since (a) v0 isn't supported anymore and (b) this message isn't going to result in a JSON-e rendering of taskcluster.yml, you'll only need the "details" that the handler requires, and it may be simpler to just send `eventData` directly rather than select and rename fields like this.